py/.github/workflows/CI.yml

```diff
@@ -0,0 +1,118 @@
+# This file is autogenerated by maturin v0.14.15
```

Comment:
I don't think this workflow is considered in Github Actions, we can remove/ignore it.